### PR TITLE
Another one.

### PR DESCRIPTION
**Pull request by bangerth**

### CODE REVIEW

File: publications-2018.bib [hunk 1]
```diff
@@ -798,4 +798,18 @@ @Article{Choo2018b
   publisher = {Elsevier {BV}},
 }
 
+@article{GDKL18,
+   author = {K. Gupta, Deepak and Keulen, Fred and Langelaar, Matthijs},
+   year = {2018},
+   month = {11},
+   pages = {},
+   title = {Design and analysis adaptivity in multi-resolution topology optimization}
```
Comment (drwells) [inline publications-2018.bib]: It looks like you are missing a comma here.

**Comment by bangerth** [comment]
Thanks for paying more attention that I apparently did! Fixed version now here!

**Comment by drwells** [comment]
I still think that it is super cool that we can use CI on latex documents like this :)

**Pull request merged**